.rubocop.yml

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
might pay to have a comment at the top of this file that this is only for this repo, not for the template?